<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Radar</class>
 <widget class="QFrame" name="Radar">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>272</width>
    <height>241</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>272</width>
    <height>241</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>272</width>
    <height>241</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Frame</string>
  </property>
  <widget class="QGraphicsView" name="vueRadar">
   <property name="geometry">
    <rect>
     <x>35</x>
     <y>20</y>
     <width>201</width>
     <height>201</height>
    </rect>
   </property>
   <property name="frameShape">
    <enum>QFrame::NoFrame</enum>
   </property>
   <property name="frameShadow">
    <enum>QFrame::Plain</enum>
   </property>
  </widget>
  <widget class="QLabel" name="coordGeo1">
   <property name="geometry">
    <rect>
     <x>115</x>
     <y>6</y>
     <width>41</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Nord</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="coordGeo2">
   <property name="geometry">
    <rect>
     <x>115</x>
     <y>220</y>
     <width>41</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Sud</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="coordGeo3">
   <property name="geometry">
    <rect>
     <x>4</x>
     <y>111</y>
     <width>31</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Est</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
   </property>
   <property name="margin">
    <number>1</number>
   </property>
  </widget>
  <widget class="QLabel" name="coordGeo4">
   <property name="geometry">
    <rect>
     <x>236</x>
     <y>111</y>
     <width>41</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>Ouest</string>
   </property>
   <property name="margin">
    <number>1</number>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
